app.organicmaps.desktop.yml

```diff
@@ -0,0 +1,50 @@
+app-id: app.organicmaps.desktop
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+command: OMaps-wrapper
```

Comment:
You need to use `OMaps-wrapper` also in the desktop file. It starts fine from CLI with `flatpak run`, but the desktop file fails to launch the app correctly.

Reply:
I solved (I think, let's see) the issue by naming the wrapper OMaps and renaming the original executable

Reply:
looks like it's fixed! https://github.com/flathub/flathub/pull/2968#issuecomment-1103541977